src/pages/hegel/reference/becoming/development.mdx

```diff
@@ -0,0 +1,562 @@
+the term Burbidge chose pollutes the thought of the logic rather than clarifying
+it.
+
+### Houlgate
+
+With the category `becoming` Hegel has reached the standpoint of Heraclitus,
+where everything flows and all is becoming (Houlgate 2022, 147). Stephen
+Houlgate contends that at this point in the _Logic_, Hegel fully affirms a
+Heraclitean vision of a ceaselessly changing reality; Hegel "is affirming a
+Heraclitean vision of sheer _becoming_" (Houlgate 2022, 147). Importantly,
+though, Houlgate points out that Hegel shows us this Heraclitean vision _by_ the
+standpoint of Parmenides, namely, the standpoint of `pure being` (although
+Parmenides himself would never leave his standpoint).
+
+The `becoming` Hegel examines, however, is not a becoming understood to occur
+_in time_. Rather, it is only the transition of `pure being` and `nothing` into
+one another. As human thinkers, we need time to think this transition; but
+`pure being` does not prove to be `nothing` in virtue of time passing, but in
+virtue of what turns out to be logically the case conceptually.
+
+> ...the _process_, described in logic, in which being proves to be becoming
+> (and then further forms of being) is not temporal ... It is the logical
+> process in which being discloses what it is in truth. Hegel's claim is thus
+> not that in time there is first being, then nothing and then becoming, but
+> that logically being itself proves to _be_ becoming (as it will later prove to
+> be substance, the Idea and space-time)(Houlgate 2022, 148).
+
+In Houlgate's reading, the categories `being` and `nothing` undergo a twofold
+transformation, namely, turning into processes of which they are themselves
+moments. `Nothing` does not simply vanish into `being`, but proves _to be_ that
+vanishing, the very transition into its opposite. In that sense, it proves to be
+the process of coming-to-be.
+
+```mermaid
+flowchart TB
+    subgraph being_as_process["<code>being</code> (<i>as process</i>)"]
+    being_a["<code>being</code>"]-->nothing_a["<code>nothing</code>"]
+    end
+    subgraph nothing_as_process["<code>nothing</code> (<i>as process</i>)"]
+    nothing_b["<code>nothing</code>"]-->being_b["<code>being</code>"]
+    end
+```
+
+In this _process of_ `nothing`, `nothing` itself is immediate and passes over
+into `being`, such that the process itself contains `nothing` and `being` as its
+two moments. The same holds for the process of `being`. "As becoming, therefore,
+being and nothing are the process of _vanishing_ itself; but, as moments of
+becoming, they are each _that which_ vanishes into the other" (Houlgate 2022,
+149). In Houlgate's understanding, then, `being` and `nothing` are each seen
+both as the _process_ of vanishing (or becoming) and as _moments_ of the
+vanishing, respectively.
+
+In Houlgate's estimate, `becoming` inaugurates a more stable thought, albeit one
+of restless instability. His point, however, is that the thought of `becoming`
```

Comment:
what about "restless stability" instead?

Reply:
I wouldn't change it because it's the terms that Houlgate uses and, given this is a reflection of his reading, I'd just stick to that. But points for being creative 🤓